Add a 'Cleaning Up' section to the user guide

digitalronin opened the pull request:
This commit adds a brief explanation of how to
remove unneeded cluster resources that the user
created while working through sections of the
guide.

The 'Cleaning Up' section is added as the last
part of the 'Deploying an app' part of the user
docs.

Commit:
Add a 'Cleaning Up' section to the user guide

This commit adds a brief explanation of how to
remove unneeded cluster resources that the user
created while working through sections of the
guide.

The 'Cleaning Up' section is added as the last
part of the 'Deploying an app' part of the user
docs.

Commit:
Add a section on deleting namespace resources

Commit:
Replace '-n' with '--namespace'

alkar commented on 02-deploying-an-app/006-cleaning-up.md:
```diff
@@ -0,0 +1,52 @@
+
+## Raising a pull request
+
+When you have finished with your sandbox namespace, and its associated ECR, please create a fork of the [environments repository][envrepo] with your environment's sub-directory of the `namespaces/cloud-platform-live-0.k8s.integration.dsd.io` directory removed, and raise a [pull request][pr] to merge your fork into master. Once merged, this will remove the YAML files which define your namespace, a ...
```
We typically give write access to the repository so that people can branch off instead of forking. Do you think we should adjust wording? Forking would just work, however, assuming no special permissions.

digitalronin replied:
You're absolutely right - I should have talked about branching, not forking. I'll change it.

jasonBirchall reviewed (approved):
Looks good. Eventually we will make the [check environments](https://github.com/ministryofjustice/cloud-platform-concourse/blob/master/pipelines/cloud-platform-live-0/main/check-environment.yaml) more aggressive but this serves as a nice stop gap 👍

alkar commented on 02-deploying-an-app/006-cleaning-up.md:
```diff
@@ -0,0 +1,52 @@
+
+When you have finished with your sandbox namespace, and its associated ECR, please create a fork of the [environments repository][envrepo] with your environment's sub-directory of the `namespaces/cloud-platform-live-0.k8s.integration.dsd.io` directory removed, and raise a [pull request][pr] to merge your fork into master. Once merged, this will remove the YAML files which define your namespace, a ...
+
+In the body of your PR, please add a note to ask the cloud platform team to manually delete both your namespace and your ECR.
```
If you haven't seen this yet, there's a [pipeline](https://github.com/ministryofjustice/cloud-platform-concourse/blob/master/pipelines/cloud-platform-live-0/main/check-environment.yaml) that reports back with the "things to clean up". We have been relying on it so far, however, I think/hope making the cleanup job part of the PR would mean we're more likely to act on it in a timely fashion 👍

digitalronin replied:
👍 Having an automated process to nag us about this stuff is great, but I think developers should be thinking about clearing up after themselves, too.

alkar reviewed (approved):
LGTM!

Commit:
Correct 'fork' to 'branch', in cleanup section

Our policy for the environments repo is for developers to create
branches with the changes they want, not forks.
